docs/source/conf.py

```diff
@@ -174,7 +174,7 @@ def system(cmd):
 extra_navbar = "<h1>dsadsa</h2>"
 
 html_sidebars = {
-	"**": ["sbt-sidebar-footer.html", "sbt-sidebar-nav.html", "sidebar-search-bs.html"]
+	"**": ["sbt-sidebar-footer.html", "sbt-sidebar-nav.html", "searchbox.html"]
```

Comment:
why was this needed?

Reply:
```suggestion
	"**": ["sbt-sidebar-footer.html", "sbt-sidebar-nav.html", "sidebar-search-bs.html"]
```
lets revert (see my comment)